piker/brokers/kraken.py

```diff
@@ -20,7 +20,7 @@
 """
 from contextlib import asynccontextmanager
 from dataclasses import asdict, field
-from typing import List, Dict, Any, Tuple, Optional
+from typing import List, Dict, Any, Tuple, Optional, AsyncIterator
```

Comment:
Drop all these data structures, we're on 3.9+ now so use the new built-in syntax.